round4/TabTracker.md

```diff
@@ -0,0 +1,109 @@
+# Chrome Extension Idea: [TabTracker]
+
+## Authors
+
+Maya Chari, Arriella Mafuta, Sid Kanderi
+
+## Problem Statement
+
+Many people use Chrome to accomplish a variety of tasks such as work, communications, entertainment, and social media. A major problem today is that we forget to close inactive tabs when using the Internet. Moreover, when there are several tabs open in the browser, navigating through different sites can be tedious. TabTracker allows us to closely monitor our usage of different web pages and mark out tabs for closure.
+
+## Target Audience
+
+Our target audience is primarily students who are juggling work for many classes at once or anyone that tends to have many tabs open on their Chrome browser. 
+
+## Description
```

Comment:
I think this is a very doable project to complete in 4 weeks. I like that there is a one sentence summary of the tab to give the user an idea of what the tab is about. Something to think about to enhance the project is to embed it within the tabs instead of having a separate extension with all the tabs listed since lots of links could look a bit cluttered